### PR TITLE
Updates mostly for real val

### PR DESCRIPTION
**Pull request by PeterMitrano**
- robot description is now namespaced as `hdt_michigan/robot_description`
- arg for whether the robot & planning description should include the Husky or not. By default it now does include the husky
- calibrate_val_currents.py now does a more complete thing, calculating motor constants and thresholds
- update val_basic_motion.py to new positions because adding the husky moved stuff around
- the URDF robot model name is now "husky" when running in the real world which is annoying :(

Tested on the robot

### CODE REVIEW

File: arm_robots/src/arm_robots/base_robot.py [hunk 1]
```diff
@@ -21,6 +20,7 @@ def __init__(self, robot_namespace: str = '', robot_description: str = 'robot_de
         TrajectoryFollower class, we do not want BaseRobot to use any MoveIt TrajectoryExeuction or rely on any trajectory
         execution services.
         """
+        robot_description = rospy.resolve_name(robot_description)
```
Comment (LemonPi) [inline arm_robots/src/arm_robots/base_robot.py:23]: why have this extra line instead of
`self.robot_description = rospy.resolve_name(robot_description)` ?

**Review by LemonPi — COMMENTED** [review]
What is the robot_description used for? We have so many names now (val, hdt_michigan, husky)... I'm getting lost about where each should be used.

Also a minor recommendation on commit names; is it possible to squash commits called `debugging`? You can try interactive rebases and setting those as fixup commits `rebase -i <remote branch or some previous commit>`. From the authorship it seems you're making them on different machines, so it's understandable though.

**Comment by PeterMitrano** [comment]
I'm happy to squash stuff, good idea.

As for the naming, yea it's a disaster, but I don't want to fix it because I don't understand where everything is used so debugging/fixing could take a while. here's what I do know:

 - the URDF is put in /hdt_michigan/robot_description and has the name `husky`. This is what you'd reference if you're working with the URDF directly which you almost never do in my experience.
 - the robot is also named husky in the SRDF which gets used implicitly by moveit stuff, again you almost never have to write out this string.
 - in gazebo the robot name is husky but the namespace is hdt_michigan. You again probably won't need to refer to the robot by name in gazebo, but the namespace needs to be hdt_michigan because that's what all the ros topics are prefaced with.
 - Val is used in the arm robots code as the class name, nowhere else AFAIK.
 - hdt_michigan is used in most of the ropic/service names

**Review by LemonPi — APPROVED** [review]
I see, yeah for now it's fine then. At some point it might make sense to replace "Val" in the code with HDTMichigan since that seems to be the name we need to interact with at the ROS level.

**Pull request merged**